### PR TITLE
chore: add new Error Fingerprinting example app

### PR DESCRIPTION
**Pull request by jmartin4563**
## Proposed Release Notes
* Added example app to demonstrate usage of `newrelic.setErrorGroupCallback`

## Links
Part of https://github.com/newrelic/node-newrelic/pull/1555

### CODE REVIEW

File: error-fingerprinting/README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  'request.uri': String,
+  'http.statusCode': String,
+  'http.method': String,
+  'error.expected': Boolean
+}
+```
+
+If the callback function returns a non-empty string, that string will be used as the Error Group name in Errors Inbox. If the callback function returns something other than a string, or an empty string, the `error.group.name` attribute will not be added to the Error.
+
+Finally, if `newrelic.setErrorGroupCallback` is invoked multiple times, the previous callback function will be overwritten, there can be only one active callback function.
```
Comment (mrickard) [inline error-fingerprinting/README.md:22]:
```suggestion
Finally, if `newrelic.setErrorGroupCallback` is invoked multiple times, the previous callback function will be overwritten. There can be only one active callback function.
```

File: error-fingerprinting/README.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+```
+
+## Example App
+
+**Running the app**
+1. `export NR_KEY=<your-key-here>`
+2. `npm run start`
+3. `npm run traffic`
+
+You can now login to the New Relic UI and observe errors generated in the `Errors Inbox` page.
```
Comment (mrickard) [inline error-fingerprinting/README.md:46]:
```suggestion
You can now log in to the New Relic UI and observe errors generated in the `Errors Inbox` page.
```

File: error-fingerprinting/index.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+'use strict'
+
+const newrelic = require('newrelic')
+const express = require('express')
+const { interactWithAPI, interactWithDatabase } = require('./helpers')
+
+const app = express()
+
+newrelic.setErrorGroupCallback(function groupErrors(errMetadata) {
```
[the remark this answers is not in the document]
Comment (jmartin4563) [reply error-fingerprinting/index.js:15]: D'oh, thank you for fixing 🙏🏻

**Review by bizob2828 — APPROVED** [review]
tested and verified this app works

**Pull request merged**